src/sass/_uswds-theme-custom-styles.scss

```diff
@@ -97,3 +97,158 @@ i.e.
     max-width: 50%;
   }
 }
+
+// Buttons
+.usa-button {
+  height:units(6);
+  padding: units(2) units(2.5);
+  &.button-icon {
+    padding-top:units(1.5);
+    padding-bottom:0.875rem;
+  }
+}
+
+.button-icon svg {
+    vertical-align: sub;
+    margin-bottom: -2px;
+  }
+
+.usa-button:focus,
+.usa-button.usa-focus {
+  outline-color: color($theme-color-primary-vivid);
+}
+
+//Primary Buttons (dark)
+.usa-button.primary-button-dark:not(:disabled) {
+  color: color($theme-text-color);
+  background-color: color($theme-color-base-lightest);
+}
+
+.usa-button.primary-button-dark.usa-button--hover:not(:disabled),
+.usa-button.primary-button-dark:hover:not(:disabled) {
+  color: color($theme-text-color);
+  background-color: color($theme-color-base-lighter);
+}
+
+.usa-button.primary-button-dark.usa-button--active:not(:disabled),
+.usa-button.primary-button-dark:active:not(:disabled) {
+  color: color($theme-text-color);
+  background-color: color($theme-color-base-light);
+}
+
+//Primary Buttons (danger)
+.usa-button--secondary {
+  background-color: color($theme-color-error);
+}
+
+.usa-button--secondary.usa-button--hover,
```

Comment:
There are some defined classes like `usa-button--hover` that are used to preview the states in the library without having to actually do that action (e.g. in the library you can see the hover styling without actually hovering over). I included them in the styling but am not sure of whether they get used outside of the purpose of displaying this styling.